### PR TITLE
Add list of operatives to operatives tab

### PR DESCRIPTION
**Pull request by JonR205**

### CODE REVIEW

File: src/factions/hearthkynSalvager.ts [hunk 1]
```diff
@@ -173,7 +174,7 @@ const theirHearthBurns: Requisition = {
...
     '\n' +
     '\n' +
-    'Once that friendly operative has incapacitated 3 enemy operatives with that Faction keyword (this does not have to be in the same battle), it’s no longer harbouring a grudge. You cannot use this Requisition if a friendly operative is already harbouring a grudge, unless you choose to renounce the grudge, in which case that friendly operative loses D6 experience points and is no longer harbouring a grudge.',
+    'Once that friendly operative has incapacitated 3 enemy Operatives with that Faction keyword (this does not have to be in the same battle), it’s no longer harbouring a grudge. You cannot use this Requisition if a friendly operative is already harbouring a grudge, unless you choose to renounce the grudge, in which case that friendly operative loses D6 experience points and is no longer harbouring a grudge.',
```
Comment (LewisRhine) [inline src/factions/hearthkynSalvager.ts]: Was this ment to be changed?

Comment (JonR205) [reply src/factions/hearthkynSalvager.ts]: No other than I don't know which is grammatically correct

Comment (LewisRhine) [reply src/factions/hearthkynSalvager.ts]: i think operatives

Comment (JonR205) [reply src/factions/hearthkynSalvager.ts]: Changed back

**Pull request merged**